sphinxcontrib/needs/layout.py

```diff
@@ -777,18 +783,6 @@ def image(self, url, height=None, width=None, align=None, no_link=False, prefix=
         # Otherwise the images gets not copied to the later build-output location
         self.app.env.images.add_file(self.need["docname"], url)
 
-        # Okay, this is really ugly.
```

Comment:
@danwos I removed this because using image link did not work as expected with this. Probably one could also remove https://github.com/useblocks/sphinxcontrib-needs/blob/d40897e023349b9e301de4cccbbf71241d12cd80/sphinxcontrib/needs/libs/html/sphinx_needs_collapse.js#L50 . I did not do it myself because I was afraid of side effects.